sys/zuse.hoon

```diff
@@ -95,6 +95,8 @@
   %-  unit                                              ::  ~: unknown
   %-  unit                                              ::  ~ ~: invalid
   (cask vase)                                           ::  marked cargo
+::
+++  turf  (list @t)                                     ::  domain, tld first
```

Comment:
You might actually be overriding the `++turf` in `hoon.hoon` with this?

Reply:
Indeed, and unfortunately, I am. I got attached to the name (and used it events/effects) before I realized. I think "turf" is a really good flw for domain, and remain attached to it. There's only one use of `^turf` in all of userspace. Maybe we can rename "utf8 to utf32"? And good catch, I meant to call this out explicitly.

Reply:
I'm in favor of renaming the utf conversion. The flw is much more appropriate here, and there's only very little code that uses it.